src/Credentials/GCECredentials.php

```diff
@@ -340,6 +345,22 @@ public static function onGce(callable $httpHandler = null)
             } catch (ConnectException $e) {
             }
         }
+
+        if (PHP_OS === 'Windows') {
+            // @TODO: implement GCE residency detection on Windows
```

Comment:
No need to include it here, just want to make sure: Is there a bug filed internally or in this repo for this todo?

Reply:
No, but that's a great idea! Filed https://github.com/googleapis/google-auth-library-php/issues/470